components/menu.tsx

```diff
@@ -41,7 +42,13 @@ type SFXMenuProperties = {
 	onClose: () => void;
 };
 
-type MenuState = 'closed' | 'sfx' | 'streak' | 'words' | 'wpm';
+type SoundMenuProperties = {
+	state: State;
+	onSetSoundEnabled: (enabled: boolean) => () => void;
```

Comment:
Once integrated into the SFX menu, please update this to match the existing naming convention: `onSetSFXSound` etc.